codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -382,10 +419,10 @@ void ensureSpaceIsAllocated(int index) {
                 // Initial size is 1/4 the number of children. Clipping the minimum at 2, which will over allocate if
                 // maxSize == 1 but if this was true we shouldn't need to re-allocate because the 1 child should get
                 // all of the available connection window.
-                stillHungry = new Http2Stream[max(2, maxSize / 4)];
+                stillHungry = new Http2Stream[max(2, maxSize >>> 2)];
```

Comment:
It seems likely that compiler/JIT will optimize this. I think it is pretty common to have bit shifts for mult/div by powers of 2 in the Netty code base.  @normanmaurer - Any thoughts?